### ISSUE TITLE
touchSensor does not work on a indexedLineSet

### ISSUE THREAD

**Issue by olafgithub** [create3000/x_ite#103]
I noticed that using a touchSensor with an IndexedLineset does not work: I see no hand cursor symbol.
With BS Contact it works fine.
I tried to make a 3D family tree viewer. Clicking on a line would show information like date of birth etc.

![ils_with_touchsensor](https://user-images.githubusercontent.com/6066508/158057014-2df53676-b097-4c08-9b56-ff6f2eb7759e.png)
With BS Contact

![ils_x_ite](https://user-images.githubusercontent.com/6066508/158057015-cbeedb94-eb76-4442-a7b0-716fb0b5c36b.png)
With X_ITE

Windows 10
Google Chrome: Version 99.0.4844.51 (Official Build) (64-bit)
X_ITE: 4.7.15

**Comment by olafgithub** [comment]
Wow.
That is very fast. I just posted about the AA on lines. Didn't even see that this one was fixed already.

When will it be available in **latest**?

Thanks,

Olaf

**Comment by create3000** [comment]
I have now published alpha version, which includes line and point picking, which is available at:

```html
<link rel="stylesheet" type="text/css" href="https://create3000.github.io/code/x_ite/alpha/dist/x_ite.css"/>
<script type="text/javascript" src="https://create3000.github.io/code/x_ite/alpha/dist/x_ite.min.js"></script>
```

I would ask you to test this to see if it works properly.

**Comment by olafgithub** [comment]
Quick check...
I can see the cursor change to a hand symbol when it is over a line. I did not write a handler yet but it looks promising.

Olaf

**Comment by olafgithub** [comment]
I checked **isOver** and **touchTime** both work fine!

Olaf

**Comment by create3000** [comment]
Have released the new version 5.0.0.
